### PR TITLE
{bio}[foss/2016b] GD 2.66

### PR DESCRIPTION
**Pull request by SethosII**
This adds a current GD with up-to-date dependencies. Might also help with https://github.com/hpcugent/easybuild-easyconfigs/issues/4394?

### CODE REVIEW

File: easybuild/easyconfigs/l/libgd/libgd-2.2.4-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'libgd'
+version = '2.2.4'
+
+homepage = 'http://libgd.bitbucket.org/'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libgd/libgd-2.2.4-foss-2016b.eb]: existing `libgd` easyconfig for 2.2.4 points to https://libgd.github.io, let's keep that consistent?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fbacc348cc5a22f7aa3c96b9514832f1 for a full test report.

**Comment by boegel** [comment]
@vanzod WIth #4394 in mind, please review/test this?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/22ce96843d99460c72b3164592e3b159 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/d8ee792d7da05b9c4e60355b181b9e37 for a full test report.

**Comment by vanzod** [comment]
lgtm too. Thanks @SethosII

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**